docs/getting-started/alerts.yaml

```diff
@@ -254,7 +254,7 @@ groups:
       message: |
         Ingesters in {{ $labels.cluster }}/{{ $labels.namespace }} ingest too many samples per second.
     expr: |
-      avg by (cluster, namespace) (rate(cortex_ingester_ingested_samples_total[1m])) > 80e3
+      avg by (cluster, namespace) (rate(cortex_ingester_ingested_samples_total[1m])) > 160e3
```

Comment:
Are these default value changed due to values changed in the jsonnet repo?

Reply:
Yes, it's from https://github.com/cortexproject/cortex-jsonnet/pull/59